lib/rules/property-blacklist/README.md

```diff
@@ -8,11 +8,13 @@ a { text-rendering: optimizeLegibility; }
  * These properties */
 ```
 
+This rule removes vendor prefixes before any checks.
+
 ## Options
 
-`array|string`: `["array", "of", "unprefixed", "properties" or "regex"]|"property"|"/regex/"`
+Type: `array|string`
```

Comment:
Missing `regex` here.

Reply:
Not quite yet. It'll fail validation until https://github.com/stylelint/stylelint/issues/3008 is done